stattutor/static/html/ctatxblock.html

```diff
@@ -0,0 +1 @@
+<iframe id="ctatxblockframe" class="stattutor" src="{tutor_html}" width="{width}" height="{height}"></iframe>
```

Comment:
By specifying an ID, you can't place more than one XBlock on the page at a time.